system/jhtree/jhtree.cpp

```diff
@@ -674,11 +674,19 @@ class DelayedCacheEntryReleaser : public IRemovedMappingCallback
     {
         if (unlikely(recordingEvents()))
         {
-            for (unsigned i1 = 0; i1 < numFixed; i1++)
-                noteEviction(fixedFileId[i1], fixedPending[i1]);
+            try
```

Comment:
Is it worthwhile to wrap each noteEviction in try...catch? The possible exception seems to be thrown by an assertex in node->getFpos(). Not sure if a failed assert means that all such calls would fail as well. If not, and if it is worth it, then wrapping each invocation may be better.

Reply:
It is an assert that is an internal error - and these notifications are for information only, so opted for the smallest change.  I guess I could move the try to noteEviction function, but I don't think it is worth the time to push etc.
I will merge as is this time